gitea/README.md

```diff
@@ -10,6 +10,19 @@ etc).
 
 **Windows users**: You must [install git](/git) first.
 
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/opt/gitea/gitea
```

Comment:
Here too, `~/.local/opt/gitea/` is sufficient, since that's the install folder that has everything related to gitea.

Reply:
Updated as described.